.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.18.1
```

Comment:
Is this right, given we seem to be using 1.16 in release.yml? 
which also makes me wonder, is 1.16 right? :P @tjdevries @efritz @chrismwendt

Reply:
should we just bump `go-version` in release.yml to 1.18 as well?

Reply:
That version has implications on lsif-go itself afaik

Reply:
@Strum355 reverted back to 1.16 for buildkite pipeline and it also supports m1 chip ;)